website/migrations/20200325005855_initial-users.js

```diff
@@ -0,0 +1,30 @@
+const path = require('path');
+const migrationName = path.basename(__filename);
+
+const users = [
+	{ email: 'john@thinkmill.com.au', hash: '$2y$10$YU4x2JB04tDN2HED9XKm6OIljAlKoZjaEx7Ip2z/mYiXjR5sT8k.e' },
+	{ email: 'dominik@thinkmill.com.au', hash: '$2y$10$4HRe4RVVJP8NcPxFh/8sWu7z/AaqW4CaYgPA3SN.tPHscQS1uNpCy' },
+	{ email: 'mike.r@thinkmill.com.au', hash: '$2y$10$UVFHfDjIcNEl5pFMKHcsQehwt5yeQrNkvQxHVN5kOqsz1JhKZ0l3G' },
```

Comment:
Even thought they are hashes I still feel funny this is in the git history

Reply:
You should change your password on first sign in. It still doesn't feel great but I think it's reasonable.